warcry_proj/README.txt

```diff
@@ -0,0 +1,22 @@
+warcry_proj README
+==================
+
+Getting Started
+---------------
+
+- Login to Vagrant
+* vagrant up
+* vagrant ssh
+
+- Ensure the bootstrap script is run to configure and install all required packages
+* sudo script/bootstrap vagrant
```

Comment:
Great idea, i think a yml file specific to each project should be created.
So in this case i would create an examples_project.yml ?